ethan-aws-finops-culture-blog

```diff
@@ -0,0 +1,86 @@
+//Ethan AWS Cost Optimisation Blog on FinOps Culture 
+
+**Looking Beyond The Technology in FinOps - How organisations need to address the People Side **
+
+FinOps, or cloud financial management, has grown significantly over the past few years, with more and more organisations prioritising it to reap critical benefits, including saving money and driving better accountability across their IT estate. To meet demand or to focus on quick wins early, organisations have steered towards the technology or tooling side first without adopting the people or process element, which drives long-term success. 
+
+At FinOps X this year, we saw a significant focus on training, certification and development of FinOps practitioners and supporting personas. JP Morgan & Co, which won FinOps team of the year and is also considered one of the largest FinOps organisations globally, has over 225 certified practitioners and training programs for the broader organisation. While establishing large teams or organisation-wide processes may be a priority for enterprise organisations, every organisation will have different sizes, preferences, spending and organisational buy-in. Whilst this blog showcases what you could do, please implement what works best. For more information, please visit the FinOps Foundation Framework: https://www.finops.org/framework/.
+
+**The FinOps Team**
+
+A solid and knowledgeable FinOps team is the cornerstone of successful FinOps Implementation across an entire organisation and its IT estate. This team comprises individuals with diverse skills and backgrounds, including technical, finance or business focus via methods of upskilling or external hiring. Here's why people matter in FinOps:
+
+
+
+* Expertise: FinOps professionals possess the expertise to understand complex cloud pricing models, identify cost optimisation opportunities, and make informed decisions. Their ability to interpret data and trends is invaluable in reducing costs effectively.
```

Comment:
Don't use the word you are defining in an explanation. Maybe change to skill set?

Reply:
Could you also add a why this important for each of them